src/components/Token/TokenBalanceCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import './TokenBalanceCard.css'
+import Icon from 'react-crypto-icons'
+import TokensPerWalletPopup from './TokensPerWalletPopup'
+import { AggregatedTokenBalance, TokenBalance } from '../../entities/Balance/types'
+
+export type TokenBalanceCardProps = React.HTMLAttributes<HTMLDivElement> & {
+  aggregatedTokenBalance: AggregatedTokenBalance
+}
+
+function formattedTokenBalance(tokenBalance: TokenBalance) {
```

Comment:
I could not import this function from '../../entities/Balance/utils', not sure why.

Reply:
It's most probably a circular dependency error. Try putting it in Wallet/utils.ts